dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -108,22 +108,6 @@ TaskFunction createCubicBezierPerfE2ETest() {
   ).run;
 }
 
-TaskFunction createCubicBezierPerfSkSlWarmupE2ETest() {
-  return PerfTestWithSkSL.e2e(
```

Comment:
Can we delete this whole class or we still need it for iOS?

Reply:
We still need it for the iOS smoke test.